.env.deploy.example

```
@@ -0,0 +1,40 @@
+# Public host handled by Caddy
+APP_HOST=api.example.com
+
+# Spring profile / runtime
+SPRING_PROFILES_ACTIVE=dev
+SERVER_PORT=8080
+TZ=Asia/Seoul
+
+# PostgreSQL
+POSTGRES_DB=insideout_db
+POSTGRES_USER=insideout_admin
+POSTGRES_PASSWORD=change-me
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)